src/reducers/isLogged.ts

```diff
@@ -0,0 +1,13 @@
+// NEED CONFIRMATION FOR ANY ACTION
+import { AnyAction } from 'redux';
+
+const loggedReducer = (state = false, action: AnyAction) => {
+    switch (action.type) {
+        case 'SIGN_IN':
```

Comment:
Do you think this is the right action name? It is not signing in a user, it is toggling the logged in state of a user.